pkg/cmd/roachtest/tests/activerecord_blocklist.go

```diff
@@ -18,29 +18,32 @@ package tests
+	`CompatibilityTest4_2#test_options_are_not_validated`:                                                                                                      "unknown",
+	`CompatibilityTest5_0#test_datetime_doesnt_set_precision_on_change_column`:                                                                                 "unknown",
+	`CompatibilityTest5_0#test_options_are_not_validated`:                                                                                                      "unknown",
+	`CompatibilityTest5_1#test_datetime_doesnt_set_precision_on_change_column`:                                                                                 "unknown",
+	`CompatibilityTest5_1#test_options_are_not_validated`:                                                                                                      "unknown",
+	`CompatibilityTest5_2#test_datetime_doesnt_set_precision_on_change_column`:                                                                                 "unknown",
+	`CompatibilityTest5_2#test_options_are_not_validated`:                                                                                                      "unknown",
+	`CompatibilityTest6_0#test_datetime_doesnt_set_precision_on_change_column`:                                                                                 "unknown",
+	`CompatibilityTest6_0#test_options_are_not_validated`:                                                                                                      "unknown",
+	`CompatibilityTest6_1#test_datetime_doesnt_set_precision_on_change_column`:                                                                                 "unknown",
+	`CompatibilityTest6_1#test_options_are_not_validated`:                                                                                                      "unknown",
+	`CompatibilityTest7_0#test_datetime_sets_precision_6_on_change_column`:                                                                                     "unknown",
+	`CompatibilityTest7_0#test_options_are_not_validated`:                                                                                                      "unknown",
+	`PostGISTest#test_point_to_json`:                                                                                                                           "unknown",
+	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_not_save_and_return_false_if_a_callback_cancelled_saving_in_either_create_or_update`: "unknown",
+	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_not_update_children_when_parent_creation_with_no_reason`:                             "unknown",
+	`TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_update_children_when_autosave_is_true_and_parent_is_new_but_child_is_not`:            "unknown",
```

Comment:
nit: these new tests are flaky, so should be in the ignoreList.

(right now they are in the blockList, which means the test will expect them to fail every time.)

Reply:
ah apologies